### DISCUSSION TITLE
每次更新完固件网络接口都会出现这么个提示

### DISCUSSION THREAD

**Discussion by smallprogram** [immortalwrt/immortalwrt#1583] [Q&A]
每次更新完固件网络接口都会出现这么个提示，官方openwrt固件就没有这种情况
![微信截图_20241113133544](https://github.com/user-attachments/assets/56a5f1f3-e901-40f4-b06b-0c76c91e40dd)

请问他修改了什么东西？

**Comment by xlighting2017** [comment]
```官方openwrt固件就没有这种情况```

### are U sure? 下面是 “官方” luci 代码片段



https://github.com/openwrt/luci/commit/b8acde7f480a18343ee1f9624b790595065613ae

![image](https://github.com/user-attachments/assets/a9b6c6e4-4ac2-42f0-8f6f-1ef9e7bf3256)

**Reply by smallprogram** [reply]
我是想知道他干了什么而不是提示什么
